### PR TITLE
Added cross-model query doc and cheat sheets

### PR DESCRIPTION
**Pull request by kian99**
Added cross-model queries doc as it previously lived only on Google Drive.

Also added the markdown and rest cheat sheets for future reference but added them to the ignore for builds.

After running make install, my go.mod and go.sum were populated, I assume it's intended to check in these with all the dependencies but if not I can remove them.

### CODE REVIEW

File: how-to/index.rst [hunk 1]
```diff
@@ -15,3 +15,4 @@ These how-to guides cover key operations and processes in JAAS.
    Candid LDAP integration <candid_ldap_integration>
    Candid Azure integration <candid_azure_integration>
    Candid ADFS integration <candid_adfs_integration>
+   Cross-Model Queries <cross_model_queries>
```
Comment (babakks) [inline how-to/index.rst:18]: Shouldn't this be added to the `url.txt` file?

**Review by babakks — COMMENTED** [review]
LGTM, with just one question/comment?

**Review by babakks — APPROVED** [review]

**Comment by alesstimec** [comment]
hmm do we need/want cheat sheets in this project?

**Comment by kian99** [comment]
@alesstimec I think the cheat sheets are handy for editors, both internal and external. But a link to the cheatsheets in the readme would also be okay. Let me know what you'd prefer.

**Pull request merged**